buddyboss/buddyboss-platform
# Fix user activity friends scope

### All Submissions:






### Changes proposed in this Pull Request:



### How to test the changes in this Pull Request:

1. Goto BuddyBoss > Settings > Activity and enable "Activity tabs"
2. User A and B are friends.
3. User B and C are friends.
4. Logged in as user C and post an activity with "My Connection" privacy.
5. Logged in as User A and view the profile of User B Go to the Connections tab and you can see user C activity.
event User A and C are not friends.



### Other information:
